source/resolve-conflicts.ts

```diff
@@ -10,8 +10,12 @@ declare module CodeMirror {
 	}
 }
 
-const select: typeof document.querySelector = document.querySelector.bind(document);
-const editor: CodeMirrorInstance = select<any>('.CodeMirror').CodeMirror;
+interface EditorLinkedDom extends HTMLElement {
+	CodeMirror: CodeMirrorInstance;
+}
```

Comment:
I don't really see the benefit of this. 

```ts
const editor = select<any>('.CodeMirror').CodeMirror as CodeMirrorInstance;
```

is good enough for one-off stuff like this.

@bfred-it ?

Reply:
That's just me conditioned to remove `any`s whenever possible 😉

Reply:
`any` and `as` sound like bandaids but I'd rather have them in these cases, e.g. https://github.com/sindresorhus/refined-github/pull/1783/commits/399a0b16595f8b426d0121b03309112c2d8d0507 and https://github.com/sindresorhus/refined-github/pull/1783/commits/44b37ce1dd68793f58770e34600a7918e28b8657

Reply:
I'll revert that change then.